docs/source/openvino/export.mdx

```diff
@@ -183,6 +183,21 @@ Besides weight-only quantization, you can also apply full model quantization inc
 optimum-cli export openvino -m openai/whisper-large-v3-turbo --quant-mode int8 --dataset librispeech --num-samples 32 --smooth-quant-alpha 0.9 ./whisper-large-v3-turbo
 ```
 
+#### Default quantization configs
+
+For some models we maintain a set of default quantization configs. To apply a default 4-bit weight-only quantization one should provide `--weight-format int4` without any additional arguments. For int8 weight & activation quantization it should be `--quant-mode int8`. For example:
+
+```bash
+optimum-cli export openvino -m microsoft/Phi-4-mini-instruct --weight-format int4 ./Phi-4-mini-instruct
+```
+
+or
+
+```bash
+optimum-cli export openvino -m openai/clip-vit-base-patch16 --quant-mode int8 ./clip-vit-base-patch16
+```
+
```

Comment:
not sure if it is a good idea, but maybe for better understanding, it also makes sense to provide a link where these default configs can be found?

Reply:
That's a good idea. For now, I've added a link to `configuration.py`. In the future a dedicated page containing all default config would be helpful.